### ISSUE TITLE
Will this library work for logging into private cloud like, US Gov cloud ?

### ISSUE THREAD

**Issue by kirankumargollapelly** [ealsur/HTTPDataCollectorAPI#8]
Hi,

I observed that the code in Collector class, located at
 **...../HTTPDataCollectorAPI/HTTPDataCollectorAPI/Collector.cs**, contains the domain name hard coded for azure public cloud as **"ods.opinsights.azure.com"**. But this is different for private clouds. 
Ex. for USGovt : "ods.opinsights.azure.us". Hence this may not work for other private clouds where this domain name differs. 

the code in Collector.cs looks like below... where the domain name is highlighted in bold.
********************************
public async Task Collect(string LogType, string JsonPayload, string ApiVersion="2016-04-01", string timeGeneratedPropertyName = null)
        {
            var utf8Encoding = new UTF8Encoding();
            Byte[] content = utf8Encoding.GetBytes(JsonPayload);

            string url = "https://" + _WorkspaceId + ".ods.opinsights.azure.com/api/logs?api-version=" + ApiVersion;
.............................................
}
********************************
This may have to be changed as configuration to make it work for all private/public clouds in azure.

Any quick solution is appreciable.

Thanks in advance.

Regards,
Kiran Kumar Gollapelly

**Comment by bbijesh** [comment]
yeah; this is a blocker for US Govt deployment
+1

**Comment by ealsur** [comment]
Good idea, I'll see if I can surface it as a config

**Comment by ealsur** [comment]
@bbijesh @kirankumargollapelly Please use package 1.0.5 https://www.nuget.org/packages/HTTPDataCollectorAPI/1.0.5

**Comment by kirankumargollapelly** [comment]
Thank you Matias Quaranta for the quick fix. 
I also made the same changes but you have committed it already... thank you very much.

Similar changes are also needed in **loganalytics-extensions-logging** package, because it in-turn uses **HTTPDataCollectorAPI**. This package is also authored by you.

I have already made the required changes in **loganalytics-extensions-logging** implementation and would like to contribute the same back to GitHub.

Can i create a pull request in **loganalytics-extensions-logging** ?

Regards,
Kiran Kumar Gollapelly
